test/config/targets.js

```diff
@@ -14,6 +14,27 @@ const targets = {
       background: {
         color: "white"
       }
+    },
+    additionalRenderingInfo: {
+      stylesheets: [
+        {
+          url:
+            "https://proxy.st-cdn.nzz.ch/context-service/stylesheet/all/test.css"
```

Comment:
can you use another url here? this is slighly misleading as we never want to load the context-service stylesheets in additionalRenderingInfo as it would result in doubling the styles on the website.